lib/hooks/pre/downcloud.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+import fs from 'fs-extra'
+import tar from 'tar'
+import { spawn } from 'child_process'
+
+const BUILD_FOLDER = './build/'
+const COZY_URL = 'downcloud.cozycloud.cc'
```

Comment:
URL only ?

Reply:
Do you mean there should be a protocol or something? The URL gets completed in various places in various ways, and this appears to be the common part.